fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogFragment.kt

```diff
@@ -39,7 +40,14 @@ class TranslationsDialogFragment : BottomSheetDialogFragment() {
     ): View = ComposeView(requireContext()).apply {
         setContent {
             FirefoxTheme {
-                TranslationsDialogBottomSheet()
+                TranslationsDialogBottomSheet(
+                    onSettingClicked = {
+                        findNavController().popBackStack()
+                        findNavController().navigate(
+                            TranslationsDialogFragmentDirections.actionGlobalToTranslationOptionsDialogFragment(),
```

Comment:
I am kinda wondering if we should be navigating from bottom sheet to another bottom sheet. Visually, this will close the first bottom sheet, and then opening the next bottom sheet. I know we do this elsewhere, but there's also a flow when we are viewing the site details where we do not have this opening and closing of the bottom sheet for navigation.

<img width="226" alt="Screen Shot 2023-10-19 at 2 13 45 AM" src="https://github.com/mozilla-mobile/firefox-android/assets/1190888/40ebf5e6-d295-4ea8-be67-e1a75e67a18f">

Reply:
We can address any changes in a follow up, but we should start asking for some UX feedback.

Reply:
+1 on asking UX feedback early and often

Reply:
I asked on Slack .

Reply:
Thanks for getting the feedback and all the work on the patch! 

I think the patch will be good to go after this is resolved or, if it looks like a bigger change, a bug is filed to smooth the transition between sheets. 🐞

Reply:
Where is the Slack conversation happening? Could you ask over in #firefox-translations?

Reply:
@iorgamgabriel @ohall-m I think it would be okay to make the experience smooth as a followup issue since we already implemented 2 different screens here and prefer an iterative approach. I think we should try to limit only implementing a single screen per issue since the amount of review comments are also growing as we try to do more.

If you try out the site permission details for `cnn.com`, you will see for the tracking protection blocked / allow navigation is already smooth. This was implemented all in a single fragment/XML - think about it as 'pages' or views that are toggled visible manually. I think we can easily do this with Compose. However, I would be curious if we could do this with using [navigation in compose](https://developer.android.com/guide/navigation/use-graph/navigate#compose) while still be in the same bottom sheet.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1861058